eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -48,7 +48,8 @@ jobs:
         VcpkgInstall: 'curl[ssl] libxml2 openssl'
         VCPKG_DEFAULT_TRIPLET: 'x64-linux'
         CmakeArgs: ' -DBUILD_TESTING=ON -DBUILD_PERFORMANCE_TESTS=ON -DRUN_LONG_UNIT_TESTS=ON -DCMAKE_BUILD_TYPE=Debug -DBUILD_CODE_COVERAGE=ON'
-        AptDependencies: 'gcovr lcov'
+        AptDependencies: 'lcov'
+        PipDependencies: 'gcovr'
```

Comment:
Why is `pip` needed now? Was `apt` not working previously?

Reply:
It is a way to get the latest `gcovr`, it is not easily available via apt.

Reply:
That is useful info, might be worth putting it in as a comment.